initfiles/bin/init_esp

```diff
@@ -33,12 +33,15 @@ SNMPID=$$
 
 killed()
 {
+    local __processID=$(cat ${PID_NAME})
     kill_process ${PID_NAME} esp 15 ${SENTINEL}
     if [[ $? -eq 1 ]]; then
         log "could not kill $component"
     else
         log "$component Stopped"
         unlock /var/lock/HPCCSystems/$component/${component}.lock
+        log "removing artifacts for process ID ${__processID} from /tmp"
+        find /tmp -name "*${__processID}" -exec rm -rf {} \;
```

Comment:
Perhaps the .java_pid<processID> files are the result of java being loaded in debug mode and thus should probably be dealt with at that level instead of a find -name -exec rm here ?
If you are going to remove a file here then why not "rm -f /tmp/.java_pid${__processID}" ?